include/JoystickHandler.h

```diff
@@ -42,6 +42,7 @@ namespace hri_safety_sense {
 
 		  ros::NodeHandle 		rosNode;
 		  ros::Publisher 		rawLeftPub, rawRightPub;
+		  bool 					useArrowsAsAxes{false};
```

Comment:
There seems to be a formatting issue here. Some extra spaces.

Reply:
Looked correct on VS code. Not sure why it shows like this on Github. Anyways fixed it.